src/hamster/overview.py

```diff
@@ -557,7 +576,8 @@ def on_report_chosen(widget, format, path):
                 webbrowser.open_new("file://%s" % path)
             else:
                 try:
-                    gtk.show_uri(gdk.Screen(), "file://%s" % os.path.split(path)[0], 0)
+                    gtk.show_uri(gdk.Screen().get_default(),
```

Comment:
```suggestion
                    gtk.show_uri(None,
```
As above,
`None` seems the standard way to say "default location"
https://lazka.github.io/pgi-docs/#Gtk-3.0/functions.html#Gtk.show_uri
https://programtalk.com/python-examples/gi.repository.Gtk.show_uri/
It does not seem possible to make a suggestion on several lines,
but of course it should be `gtk.show_uri(None, "file://%s" % path, gdk.CURRENT_TIME)`